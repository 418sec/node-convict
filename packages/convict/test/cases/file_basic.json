{
  "foo": "Yoyodyne",
  "port": 8080
}
