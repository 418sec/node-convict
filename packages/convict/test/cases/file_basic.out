{
  "foo": "Yoyodyne",
  "port": 8080,
  "max_count": 800
}
